PocketCastsTests/Tests/Player/Transcripts/TranscriptModelTests.swift

```diff
@@ -0,0 +1,124 @@
+import XCTest
+
+@testable import podcasts
+
+final class TrasnscriptModelFilterTests: XCTestCase {
+
+    func testVTT() throws {
+        let transcript = """
+        WEBVTT - Flying Too High: AI and Air France Flight 447
```

Comment:
What do you think about changing the words here randomly? To avoid any copyright issues.

Reply:
Done here: https://github.com/Automattic/pocket-casts-ios/pull/1870/commits/3bd9fb6c72e069c544fb27de2f991cf11e258632